Test Reviewnb.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>Should have used a Fortran kernel - https://t-redactyl.io/blog/2020/01/using-jupyter-to-learn-fortran.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rkamalf/test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>